content/docs/guides/debugging.md

```diff
@@ -30,6 +30,22 @@ You can install the most common dependencies through the package manager:
 
     sudo apt-get install gdb-multiarch openocd
 
+### Fedora
+
+Fedora's `gdb` package comes compiled with multi arch support:
+  
+    dnf install gdb openocd
+
+To use the picprobe as a debugger you must compile openocd from source using Raspberry Pi's openocd. [Getting Started with Raspberry Pi Pico](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) Appendix A has build instructions we can modify:
```

Comment:
Spelling?

Reply:
Thank you for pointing these two out. I've amended my commit to correct the spelling and the extra whitespace.